core/modules/image/image.tokens.inc

```diff
@@ -0,0 +1,327 @@
+<?php
+/**
+ * @file
+ * Builds placeholder replacement tokens for image styles.
+ */
+
+/**
+ * Implements hook_token_info().
+ */
+function image_token_info() {
+  $styles = image_styles();
+  $styles_tokens = array();
+  $image_style = array();
+
+  foreach ($styles as $style => $desc) {
```

Comment:
Nit-picking, but can we please try to avoid shortened words for variable names? ...not everything is straight-forward for non-English speakers.

```suggestion
  foreach ($styles as $style => $description) {
```

Reply:
done